BezierSwift/Sources/Components/Switch/BezierSwitchControl.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  BezierSwitchControl.swift
+//  BezierSwift
+//
+//  Created by 구본욱 on 10/23/24.
+//
+
+import SwiftUI
+
+public struct BezierSwitchControl: View {
+  @Binding public var isOn: Bool
+
+  public init(isOn: Binding<Bool>) {
+    self._isOn = isOn
+  }
+
+  public var body: some View {
+    ZStack(alignment: self.alignment) {
+      RoundedRectangle(cornerRadius: 100.0, style: .circular)
+        .foregroundColor(self.backgroundColor)
+        .frame(width: 44, height: 28)
+
+      Circle()
+        .foregroundColor(BezierColor.fgAbsoluteWhiteDark.color)
+        .frame(width: 22, height: 22)
+        .applyBezierShadow(.shadow2)
+        .padding(3)
+    }
+    .onTapGesture {
+      self.isOn.toggle()
+    }
+    .animation(.easeInOut(duration: 0.3), value: self.isOn)
+    .padding(2)
+    .compositingGroup()
```

Comment:
```suggestion
    .applyDisabledStyle()
```
Disable을 고려해서 요 모디파이어를 써주시면 좋을 것 같아요